source/plugin/internals/implementation.rst

```diff
@@ -65,5 +65,5 @@ Example Using Gradle
 .. code-block:: groovy
 
     dependencies {
-        compile 'org.spongepowered:spongeforge:1.12.2-2705-7.1.0-BETA-3442:dev'
+        compile 'org.spongepowered:spongeforge:1.12.2-2705-7.1.0-RC3442:dev'
```

Comment:
Same as above.

The versioning change for SF happened with RC3484

Reply:
Corrected